pages/TheExplorationLibrary.html

```diff
@@ -36,7 +36,7 @@ <h2>
     the gallery shows only mathematics explorations.
   </p>
 
-  <img src="static/images/libraryCategoryHighlighted.png" width="100%">
+  <img src="static/images/libraryCategoryHighlighted.png" width="100%" alt="The navigation bar has mathematics as the selected category, the page displays a view of 8 explorations, 4 in each row, pertaining to the category.">
```

Comment:
Set the context please -- I'm not sure, from reading the description, what this is describing. Also consider breaking this into shorter sentences (currently you have a run-on sentence).